conf/openmetadata.yaml

```diff
@@ -225,6 +225,8 @@ elasticsearch:
 eventMonitoringConfiguration:
   eventMonitor: ${EVENT_MONITOR:-prometheus}  # Possible values are "prometheus", "cloudwatch"
   batchSize: ${EVENT_MONITOR_BATCH_SIZE:-10}
+  pathPattern: ${EVENT_MONITOR_PATH_PATTERN:-["/*", "/api/v1/tables/*", "/api/v1/health-check"]}
```

Comment:
if we have "/*" then mapping for /api/v1/tables/*", "/api/v1/health-check is by default there , remove /* 
@harshach any other endpoint we want to monitor